README.md

```diff
@@ -69,6 +69,8 @@ See list above for supported database drivers.
 ### API tools
 
 * ✅ [`graphql`](https://www.npmjs.com/package/graphql) 16.x
+* ✅ [`xml2js`](https://www.npmjs.com/package/xml2js) 0.6.x, 0.5.x, ^0.4.18
+* ✅ [`fast-xml-parser`](https://www.npmjs.com/package/fast-xml-parser) 4.x
```

Comment:
Shall we move this in a new category or keep them under API tools?
- Data serialization tools
- Data serialization/deserialization
- Data parsing